### PR TITLE
Polishing new lifted irk

### PR DESCRIPTION
**Pull request by zanellia**
Removed redundant code and unnecessary BLASFEO <-> COLMAJ conversions.

### CODE REVIEW

**Comment by FreyJo** [comment]
@zanellia besides what we talked about (calling `impl_ode_fun_jac_x_xdot_u` instead of `impl_ode_fun` + `impl_ode_jac_x_xdot_u` if `update_sens == true`) this looks good to me!

**Comment by zanellia** [comment]
This should be ready to merge. @dkouzoup, @roversch can you have a look too?

File: acados/sim/sim_irk_integrator.c [hunk 1]
```diff
@@ -311,7 +311,7 @@ int sim_irk_workspace_calculate_size(void *config_, void *dims_, void *opts_)
 
 
 
-static void *sim_irk_workspace_cast(void *config_, void *dims_, void *opts_, void *raw_memory)
+static void *sim_irk_cast_workspace(void *config_, void *dims_, void *opts_, void *raw_memory)
 {
```
Comment (dkouzoup) [inline acados/sim/sim_irk_integrator.c]: @zanellia  the naming convention is first the struct and then what to do. Therefore "sim_irk_workspace_cast" was correct. Can you set it back please? Similarly to "ocp_qp_in_memory_calculate_size" (used to be "ocp_qp_in_calculate_memory_size") etc

File: examples/c/wind_turbine_nmpc.c [hunk 2]
```diff
@@ -172,7 +181,10 @@ void ext_fun_h1(void *fun, ext_fun_arg_t *type_in, void **in, ext_fun_arg_t *typ
 int main()
 {
     // _MM_SET_EXCEPTION_MASK(_MM_GET_EXCEPTION_MASK() & ~_MM_MASK_INVALID);
-
+    feenableexcept(FE_INVALID   | 
+                   FE_DIVBYZERO | 
+                   FE_OVERFLOW  | 
+                   FE_UNDERFLOW);
```
Comment (dkouzoup) [inline examples/c/wind_turbine_nmpc.c]: what's this? :)

**Pull request merged**